src/style.css

```diff
@@ -1,3 +1,73 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  background-color: #fff;
+}
+
+@media (min-width: 1024px) and (max-width: 1200px) {
+  header {
+    height: 60px;
+  }
+}
+
+header {
+  height: 60px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: baseline;
+  position: relative;
+}
+
+ul {
+  list-style: none;
+  display: flex;
+  align-content: stretch;
+  flex-direction: row;
+  margin-right: 50px;
+}
+
+li {
+  margin: 20px;
+  align-items: center;
+  text-align: center;
+  box-sizing: border-box;
+}
+
+.is-active {
+  color: #00acdc;
+  text-decoration: none;
+}
+
+.is-active::after {
+  position: relative;
+  width: 37px;
+  height: 4px;
+  content: "";
+  border-radius: 8px;
+  background-color: #00acdc;
+  top: 20px;
+  display: block;
+}
+
+.item-4 {
+  color: black;
+  text-decoration: none;
+}
+
+.item-4:hover {
+  color: #00acdc;
+}
```

Comment:
```suggestion
```
style .nav__link. each item should have hover. on design it is just example for fourth item

Reply:
Not fixed from the previous review